Sprint 08

Peppa-Akk opened the pull request:
- Configurate target;
- Made Launch Screen;
- Made Main Screen.

Peppa-Akk committed:
Sprint 8:
- Configurate target;
- Made Launch Screen;
- Made Main Screen

Peppa-Akk committed:
Added Mock Data

So1ovyov commented on ImageFeed/ImagesList/ImagesListViewController.swift:
```diff
@@ -0,0 +1,74 @@
+
+import UIKit
+
+class ImagesListViewController: UIViewController {
```
Тут можно добавить [final](https://cocoacasts.com/swift-fundamentals-how-to-use-swift-final-keyword)
и тем самым предотвратить переопределение метода, свойства или нижнего индекса, пометив их как final
Также подробный ответ можешь прочитать [здесь](https://stackoverflow.com/questions/49405792/while-developing-for-ios-can-viewcontroller-classes-be-marked-final-if-we-do-not)

So1ovyov commented on ImageFeed/ImagesList/ImagesListViewController.swift:
```diff
@@ -0,0 +1,74 @@
+}
+
+// MARK: - Extension DataSource
+extension ImagesListViewController: UITableViewDataSource {
```
Круто что разделил реализации протоколов в расширениях, это очень удобно.

So1ovyov commented on ImageFeed/ImagesList/ImagesListViewController.swift:
```diff
@@ -0,0 +1,74 @@
+}
+
+// MARK: - Extension UITableViewDelegate
+extension ImagesListViewController: UITableViewDelegate {
```
🔥

Peppa-Akk committed:
Fix:
- Size of button like
- Delete point "Supported interface orientations (iPhone)"
- Delete points "Supported interface orientations (iPad)"
